From 8e54d665c699b6dfdd5454aaa7fcde4ec5e6c5ee Mon Sep 17 00:00:00 2001
From: Clark Williams <williams@redhat.com>
Date: Tue, 4 Oct 2011 13:02:23 -0500
Subject: [PATCH 846/858] [mrg] disable NO_HZ by default

CONFIG_NO_HZ is compiled into the MRG kernel but is disabled by default.

Signed-off-by: Clark Williams <williams@redhat.com>
Signed-off-by: John Kacur <jkacur@redhat.com>Signed-off-by: Clark Williams <williams@redhat.com>

Conflicts:
	kernel/time/tick-sched.c
(cherry picked from commit 70876438845ce54e92b7a3025059120797efc514)

Signed-off-by: Clark Williams <williams@redhat.com>
---
 kernel/time/tick-sched.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kernel/time/tick-sched.c b/kernel/time/tick-sched.c
index 626b320f..f760c57 100644
--- a/kernel/time/tick-sched.c
+++ b/kernel/time/tick-sched.c
@@ -152,7 +152,7 @@ static void tick_sched_handle(struct tick_sched *ts, struct pt_regs *regs)
 /*
  * NO HZ enabled ?
  */
-int tick_nohz_enabled __read_mostly  = 1;
+int tick_nohz_enabled __read_mostly  = 0;
 
 /*
  * Enable / Disable tickless mode
-- 
1.8.3.1

